app/src/main/java/org/sagebionetworks/research/mpower/tracking/TrackingMenuFragment.java

```diff
@@ -164,26 +175,53 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         trackingViewModel.reportLiveData().observe(this, trackingReports -> {
             // No-op needed, we use these reports passively when the user taps a tracking task.
         });
+
+        trackingAdatper = createAdapter(TRACKING_ICONS, TRACKING_LABELS);
+        measuringAdapter = createAdapter(MEASURING_ICONS, MEASURING_LABELS);
     }
 
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup group, @Nullable Bundle savedInstanceState) {
         View result = inflater.inflate(R.layout.tracking_menu_fragment, group, false);
         this.unbinder = ButterKnife.bind(this, result);
+
+        trackingLayout = new LinearLayoutManager(
```

Comment:
Major refactor of TrackingMenuFragment to use a RecyclerView.  This code was written by one of your interns so I think it benefited from an upgrade in general.